code/src/components/SingleThought.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import Moment from 'react-moment';
+import 'moment-timezone';
+
+export const SingleThought = ({ message, hearts, createdAt, _id }) => {
+  const [singleThoughthearts, setSingleThoughtHearts] = useState(hearts);
+
+  // Function for likes:
+  const handleLikeBtnClick = () => {
+    // Post a like to the API
+    fetch(`https://happy-thoughts-technigo.herokuapp.com/thoughts/${_id}/like`, {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' }
+    })
+      // Then increase the local like count with one:
+      .then(setSingleThoughtHearts(hearts += 1))
+  };
+  // Extract info for a single thought in the list of all thoughts in the API & display in a div:
+  return (
+    <>
+      <p className="message">{message}</p>
+      <div className="cardFooter">
+        <div>
+          <button
+            hearts={singleThoughthearts}
+            type="button"
+            onClick={handleLikeBtnClick}
+            className={singleThoughthearts === 0 ? 'heartBtn' : 'heartBtn pink'}>
+            ❤️
+          </button>
+          &nbsp; x {singleThoughthearts}&nbsp;
+        </div>
```

Comment:
Does this give the heart count? Send me a line, haven't seen it before (I think).

Reply:
This is just to insert a space between the heart button and the x, and between the x and the actual count :)